### PR TITLE
update premix recycling strings for 10_4_0_pre1

### PR DESCRIPTION
**Pull request by zhenhu**
Update strings to use premix libraries produced with the new PREMIX format.

### CODE REVIEW

**Comment by zhenhu** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -608,20 +608,20 @@ def identitySim(wf):
...
-    'CMSSW_10_0_0_pre2-100X_mc2017_realistic_v1-v1',     # 13 - 13 TeV samples with GEN-SIM from PhaseI upgrade
-    'CMSSW_10_3_0_pre5-PU25ns_103X_mc2017_realistic_v1-v1',     # 14 - fullSim PU 25ns UP17 premix
-    'CMSSW_10_3_0_pre5-PU25ns_103X_upgrade2018_realistic_v7-v1',  #15 - fullSim PU 25ns UP18 premix
-    'CMSSW_10_3_0_pre5-103X_upgrade2018_realistic_v7-v1',  #16 - GENSIM input 2018
-    'CMSSW_10_3_0_pre5-103X_mc2017_realistic_v1_FastSim-v1',    # 17 - fastSim MinBias for mixing UP17
-    'CMSSW_10_3_0_pre5-PU25ns_103X_mc2017_realistic_v1_FastSim-v1',# 18 - fastSim premixed MinBias UP17
+    'CMSSW_10_0_0_pre2-100X_mc2017_realistic_v1-v1',     # 13 - GENSIM input for 2017 fullSim premix workflows
+    'CMSSW_10_4_0_pre1-PU25ns_103X_mc2017_realistic_v1-v1',     # 14 - fullSim PU 25ns UP17 premix library
+    'CMSSW_10_4_0_pre1-PU25ns_103X_upgrade2018_realistic_v7-v2',  # 15 - fullSim PU 25ns UP18 premix library
+    'CMSSW_10_3_0_pre5-103X_upgrade2018_realistic_v7-v1',  # 16 - GENSIM input for 2018 fullSim premix workflows
```
Comment (makortel) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: Isn't this GEN-SIM (and on line 619) used as an input for other workflows than premix library as well?

Comment (zhenhu) [reply Configuration/PyReleaseValidation/python/relval_steps.py]: These GEN-SIM strings are used only in [L741-L762](https://github.com/cms-sw/cmssw/blob/31d700d50188597b295a756163698afe6a7c4a19/Configuration/PyReleaseValidation/python/relval_steps.py#L741-L762), which are inputs for the premix workflows. 

Other 2017 and 2018 noPU/PU workflows are "upgrade" workflows, whose use strings in line [L2735-L2738](https://github.com/cms-sw/cmssw/blob/31d700d50188597b295a756163698afe6a7c4a19/Configuration/PyReleaseValidation/python/relval_steps.py#L2735-L2738)

Comment (makortel) [reply Configuration/PyReleaseValidation/python/relval_steps.py]: Right, I forgot that the "upgrade" non-premixing workflows got their input datasets from the other source. Thanks!

**Comment by zhenhu** [comment]
please test workflow 250202.0,500202.0,250202.17,250202.18,250402,250402.17

test premix inputs for 2015 50ns (500202.0), 2016 25ns (250202.0), 2017 (250202.17), 2018 (250202.18), fastSim2016 (250402), fastSim2017 (250402.17)

**Comment by zhenhu** [comment]
+1

**Comment by fabiocos** [comment]
+1

@zhenhu thanks, we should clean the IB with this. @kpedro88 I move forward with the integration, it is not upgrade specific, please sign it in case for reference

**Comment by fabiocos** [comment]
merge

**Comment by kpedro88** [comment]
+upgrade

**Pull request merged**